### DISCUSSION TITLE
[Laravel] Should pest delete temporary database files after running in parallel?

### DISCUSSION THREAD

**Discussion by noah1400** [pestphp/pest#1330] [Q&A]
When I am running pest in parallel using sqlite as my database driver it creates a temporary database file for each process. If I call my database file `database.sqlite` and  it creates files `database.sqlite_test_{process no.}`. Shouldn't these files be deleted after the run is finished?

**Comment by schonhoff** [comment]
Hi,
I'm not using sqlite but MySQL and the databases created on the parallel testing are still there after running the tests. If you start the tests again they will be deleted (truncated?). On a PHPUnit run (before I switched to PEST) the same thing happened.

It would be a nice feature of course :-)
